### ISSUE TITLE
Too much padding in section switches

### ISSUE THREAD

**Issue by mueller-ma** [openhab/openhab-android#1602]
[Originally posted](https://github.com/openhab/openhab-android/commit/dc139517abffad8e86d0b10db2710e1b0ac0ecda#commitcomment-35421244) by @jmau2002:

This looks good for single buttons: 
![SmartSelect_20191009-095115_openHAB Beta](https://user-images.githubusercontent.com/6906979/66462102-a9f7a680-ea7a-11e9-97c9-58112b0f41b1.jpg)

But for Player items the icons are so large that the last one is not shown anymore (tested on S10). 
![SmartSelect_20191009-095154_openHAB Beta](https://user-images.githubusercontent.com/6906979/66462110-afed8780-ea7a-11e9-93c5-8f0170a8b056.jpg)

Anyway to influence the size of the icons? Maybe reduce paddingStart/End?

**Comment by mueller-ma** [comment]
Comparison between old and new:
![s1](https://user-images.githubusercontent.com/22525368/66585988-23c98600-eb88-11e9-886e-e82c8d868193.png)
![s2](https://user-images.githubusercontent.com/22525368/66585989-23c98600-eb88-11e9-9d43-dc9bebad0027.png)

**Comment by maniac103** [comment]
The current style is whatever is default for a material button ... from the comparison, we'll need to override fontSize (maybe also fontFamily), allCaps (turn it off) and paddingHorizontal.

**Comment by mueller-ma** [comment]
> allCaps (turn it off)

Basic UI also displays section switches in all caps, so it's fine to do so in the app as well.

**Comment by mueller-ma** [comment]
The buttons should use `colorAccent` as color, not `colorPrimary`.

**Comment by mueller-ma** [comment]
I started with a style, but didn't find the right background color and margin attributes yet:
````
    <style name="SectionSwitch" parent="Widget.MaterialComponents.Button.OutlinedButton">
        <item name="android:textAllCaps">true</item>
        <item name="android:textAppearance">?android:attr/textAppearanceMedium</item>
        <item name="android:textColor">?colorAccent</item>
    </style>
````

**Comment by maniac103** [comment]
Fixed, see #1606 ;-)

Main issue was - besides the large, wide and bold font - a minWidth being set on the buttons.

**Comment by jmau2002** [comment]
Looks good to me. Thanks.
